packages/bloc/pubspec.yaml

```diff
@@ -1,20 +1,23 @@
 name: bloc
 description: A predictable state management library that helps implement the BLoC (Business Logic Component) design pattern.
-version: 6.0.3
+version: 6.1.0-nullsafety
 repository: https://github.com/felangel/bloc/tree/master/packages/bloc
 issue_tracker: https://github.com/felangel/bloc/issues
 homepage: https://github.com/felangel/bloc
 documentation: https://bloclibrary.dev
 
 environment:
-  sdk: ">=2.6.0 <3.0.0"
+  sdk: ">=2.10.0-99.0.dev <3.0.0"
```

Comment:
It is safest to leave this at `<2.10.0` for now - the feature might still change and if you did accidentally publish this package in this state that limits the potential negative impact.